netlify-example/imageLoader.js

```diff
@@ -45,23 +51,23 @@
       throw 'unknown imageId';
     }
 
-    var image = {
-      imageId: imageId,
+    let image = {
+      imageId,
       minPixelValue: 0,
       maxPixelValue: 257,
       slope: 1.0,
       intercept: 0,
       windowCenter: 127,
       windowWidth: 256,
       render: cs.renderGrayscaleImage,
-      getPixelData: getPixelData,
+      getPixelData,
       rows: height,
       columns: width,
-      height: height,
-      width: width,
+      height,
+      width,
       color: false,
       columnPixelSpacing: 0.8984375,
-      rowPixelSpacing: 0.8984375,
+      rowPixelSpacing: 0.3,
```

Comment:
Is this the only _real_ change for this file?

I'm not sure if we have any tests that use this, but it might be worth adding a new image that actually has non-square pixel spacing? Something semantic, so it's more obvious what the difference between each is, so we can better use them and observe behavior.

For example:
- keep: `imageId === 'example://1` 
- add: `imageId === 'example://non-square-spacing'`

Also, some of these `let`s can be `const`s

Reply:
Note skewed demo image with new spacing:

![image](https://user-images.githubusercontent.com/5797588/64617000-9b4e9e80-d3ab-11e9-8a1c-baeee6f3c3d0.png)

Reply:
I've changed that in this branch only for tests purposes.
The PR was still a draft.
I've just reverted it back to the original values.